packages/devtools_app/lib/src/timeline/timeline_flame_chart.dart

```diff
@@ -261,11 +261,11 @@ class FullTimelineFlameChartCanvas extends FlameChartCanvas<FullTimelineData> {
       final FullTimelineEventGroup group = data.eventGroups[groupName];
       // Expand rows to fit nodes in [group].
       assert(rows.length == currentRowIndex);
-      expandRows(rows.length + group.eventsByRow.length);
+      expandRows(rows.length + group.rows.length);
 
-      for (int i = 0; i < group.eventsByRow.length; i++) {
-        final row = group.eventsByRow[i];
-        for (var event in row) {
+      for (int i = 0; i < group.rows.length; i++) {
+        final rowEvents = group.rows[i].events;
+        for (var event in rowEvents) {
```

Comment:
nit: variable name rowEvents  doesn't really add any value. consider inlining it.

Reply:
done